Makefile

```diff
@@ -1,42 +1,46 @@
-.DEFAULT-GOAL := help
+CRYSTAL=/usr/bin/crystal
+CRYSTAL_FLAGS=--release
 
-# Variables
-PROJECT_NAME?=$$(git remote -v | head -n1 | awk '{print $$2}' | sed 's/.*\///' | sed 's/\.git//')
-CWD=$(shell pwd)
-VERSION_NUMBER?=$$(docker run -e TYPE=semver -v $(CWD):/repo codacy/ci-git-version:latest)## lazy. run with VERSION_NUMBER=x to prevent slow runs
+all: clean fmt test build docker_build ## clean and produce target binary and docker image
```

Comment:
Since this is the default target it might not very safe to have it make clean

Reply:
removed